### PR TITLE
Update Wootric integration

### PR DESCRIPTION
**Pull request by diegoserranoa**
**What does this PR do?**
Updates the Wootric integration with some new variables.

**Are there breaking changes in this PR?**
No

**Has this been tested end-to-end? Please provide screenshots on how the fix now populates in the end tool. If not, what was done to test?**
How do you suggest to test this? Ideally, there would be a way to generate a similar `analytics.js` that is used in production with the new tests and then I could reference that file in an html and test the changes. I haven't found a way to do that though.

**Any background context you want to provide?**
Integration needs to be updated to send the user_id to Wootric, to be used in case we need to send back information to Segment. 

**Is there parity with the server-side/android/iOS integration (if applicable)?**
No parity with Android & iOS integrations.

**Does this require a metadata change? If so, please link the PR from https://github.com/segmentio/destination-scripts.**
I'm not sure what this means. Going to https://github.com/segmentio/destination-scripts returns a 404

**What are the relevant tickets?**

**Link to CC ticket**

**List all the tests accounts you have used to make sure this change works**

**Helpful Docs**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5,18 +5,21 @@
 "@babel/code-frame@7.0.0-beta.51":
```
Comment (fathyb) [inline yarn.lock]: Looks like these changes are not related to the functionality, could you remove them?

**Comment by diegoserranoa** [comment]
Hello! I have a question: before the integration was moved to this mono-repo, we created a PR with some changes. But when that PR was merged, some bug was introduced and we asked for it to be rolled back. We were told it was rolled back, but those changes from that commit don't appear as changes here (meaning that they stayed in the code). This is the commit I am referring to https://github.com/segment-integrations/analytics.js-integration-wootric/commit/48f7c8c63cceef390949827fdd46d4c1a6bf98e5

Changes we added were to pass the `userId`. Just wanted to clarify that because those changes should be considered too before merging this.

**Comment by diegoserranoa** [comment]
Hi, could I get a review here? 🙏

**Comment by diegoserranoa** [comment]
Hi @ccnixon could I get a review here? Sorry if you're not the person I should be pinging. If not, who should I contact to get a review here?

**Comment by srthurman** [comment]
@diegoserranoa hey Diego! Sara here :) Per our last email convo, we won't be able to prioritize the review and merging here at the moment. We'll let you know as we have an update for you, but I don't have a timeline on that just yet.

**Comment by diegoserranoa** [comment]
@srthurman Hi Sara! Thanks for replying. Isn't there any possibility to get this reviewed sooner? We're working with Chris and Segment Sources team and it would really be great if we could coordinate to get this reviewed sooner.

I'm sorry for pushing things, but we really need to get this out to get things going. 🙏
